Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Input: ");

```

Comment:
Prompt message could have asked for numeric value.

Reply:
Done